.github/workflows/ci.yml

```diff
@@ -34,9 +57,9 @@ jobs:
             python: "3.8"
             toxpy: "38"
           - test_type: packages-dynamic
-            ansible: "2.9"
-            python: "3.7"
-            toxpy: "37"
+            ansible: "3.0"
+            python: "3.9"
+            toxpy: "39"
```

Comment:
I'm not totally sure if the minor differences in version numbers between these configs is intentional or not, and I didn't cross check it with nightly and release.

Reply:
I think, this is to sprinkle some sparse entries into a huge parameter matrix.
However, python and toxpy must match.

Reply:
Exactly.

I suspect that  @fao89 forgot to update ci.yml when he updated pull_request.yml in #545